code/game/machinery/experimental_cloner/experimental_cloner_computer.dm

```diff
@@ -0,0 +1,162 @@
+/// Computer which starts the experimental cloning process
+/obj/machinery/computer/experimental_cloner
+	name = "experimental cloner control console"
+	desc = "It scans DNA structures."
+	circuit = /obj/item/circuitboard/computer/experimental_cloner
+	icon_screen = "crew"
+	icon_keyboard = "med_key"
+	light_color = LIGHT_COLOR_GREEN
+	interaction_flags_click = ALLOW_SILICON_REACH
+	/// Our current stored cloning record
+	var/datum/experimental_cloning_record/stored_record
+	/// Scanner we save a test subject from
+	var/obj/machinery/experimental_cloner_scanner/input
+	/// Pod we print someone into
+	var/obj/machinery/experimental_cloner/output
+
+/obj/machinery/computer/experimental_cloner/Initialize(mapload, obj/item/circuitboard/circuit)
+	. = ..()
+	if (!mapload)
+		return
+
+	var/list/stuff_in_range = view(7, src)
```

Comment:
doing this in Initialize is a little suspect to me, it's broken shit (missing typically the cloning pods) when I've tested locally but I can't really explain WHY, if I had to guess something about lighting but I'm really not sure. LateInitialize should? avoid those problems if they are what I think (also avoids working with uninit'd atoms)